site/docs/v1/tech/apis/registrations.adoc

```diff
@@ -127,7 +127,9 @@ include::docs/v1/tech/apis/_user-registration-response-body.adoc[]
 
 == Update a User Registration
 
-This API is used to update a User Registration for a User. This API is used to update the User's registration for a specific Application. You must specify the User Id for the User that is updating their information on the URI. The Id of the Application the User is updating their information for is sent in the request body. This API doesn't merge the existing User Registration and your new data. It replaces the existing User Registration with your new data.
+:capitalized_object_name: User Registration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```

Comment:
Will this generic explanation cover this API? There isn't a registration Id, so the unique Id is the `userId` + `applicationId`.

Reply:
Changed to handle this composite key.